src/NSwag.CodeGeneration.CSharp/Templates/ClientTemplate.tt

```diff
@@ -113,10 +113,20 @@ public partial interface I<#=Model.Class#>
     }
 
 <#}#>
+    partial void UpdateJsonSettings(ref Newtonsoft.Json.JsonSerializerSettings settings);
     partial void PrepareRequest(<#=Model.HttpClientType#> client, System.Net.Http.HttpRequestMessage request, string url);
     partial void PrepareRequest(<#=Model.HttpClientType#> client, System.Net.Http.HttpRequestMessage request, System.Text.StringBuilder urlBuilder);
     partial void ProcessResponse(<#=Model.HttpClientType#> client, System.Net.Http.HttpResponseMessage response);
 
+    Newtonsoft.Json.JsonSerializerSettings NewJsonSettings()
```

Comment:
Rename to CreateJsonSerializerSettings (a method must contain a verb, but new is not a verb)